app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -957,6 +966,33 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
         }
     }
 
+    private fun hasCameraPermission(context: Context): Boolean {
```

Comment:
I'd move this function here to `AndroidPermissionsHelper` class in order to centralize all the permissions stuffs. 

We could also move the `canWriteToExternalStorage()` and `checkWritingPermission()` from the `ImageHelper` class to `AndroidPermissionsHelper` - It can be done on other PR though.

Reply:
@filipedelimabrito I'll do both the things on this PR only.